backend/src/api/cron/cronUpdateInfo.js

```diff
@@ -0,0 +1,77 @@
+const cron = require('node-cron');
+const UserDB = require('../users/user.model');
+const { TransactionModel } = require('../transactions/transaction.model');
+const monthReportModel = require('./monthReport.model');
+
+const updateInfo = () => {
+  //cron.schedule('*/10 * * * * *', async () => {
+  cron.schedule('0 0 1 * *', async () => {
+    const users = await UserDB.find();
+
+    //await TransactionModel.deleteMany({ type: "INCOME" });
+    //await UserDB.deleteMany();
+    // await UserDB.insertMany();
+    //await monthReportModel.deleteMany();
+
+    await Promise.all(
+      users.map(async user => {
+        const income = user.totalSalary + user.passiveIncome;
```

Comment:
Could I ask you to split logic, mentioned in the task by separate functions?
Like `await incremetUserBalance(user);` etc. It would be much more easier to look through the logic in code.